CONTRIBUTION.md

```diff
@@ -0,0 +1,57 @@
+# Contribution guidlines
+## Preamble
+These guidelines are intended as "guidelines" to how an PR To Dire's Buildinggadgets should be structured. 
+Bear in mind that these are only guidelines and don't necessarily apply in each and every case, but if you choose not to follow a specific Part of this specification, you must be able to reason out why, if asked to do so.
+## PR structure
+### General Notes
```

Comment:
Should this one be removed? I don't add another one after Code Guidelines too...